contessa/consistency_checker.py

```diff
@@ -136,19 +178,26 @@ def do_consistency_check(
             )
         right_result = self.run_query(self.right_conn, right_sql, context)
 
+        valid, passed, failed = self.compare_results(left_result, right_result)
+
         return {
             "check": {"type": method, "description": "", "name": "consistency",},
-            "status": "valid"
-            if self.compare_results(left_result, right_result)
-            else "invalid",
+            "status": "valid" if valid else "invalid",
+            "passed": passed,
+            "failed": failed,
             "left_table_name": left_check_table.fullname,
             "right_table_name": right_check_table.fullname,
             "time_filter": time_filter,
             "context": context,
         }
 
     def compare_results(self, left_result, right_result):
-        return set(left_result) == set(right_result)
+        left_set = set(left_result)
+        right_set = right_result
```

Comment:
Doesnt this need to be a set as well?

Reply:
yes, thx

Reply:
interestingly tests passed because set.intersection works also with lists